README.md

```diff
@@ -18,20 +18,23 @@ Generate interactive API documentations from Swagger files. [Try our Demo](https
 
 ## Table of Contents
 
-- [Getting Started](#getting-started)
-  - [From a CDN](#from-a-cdn)
-  - [With Vue.js](#with-vuejs)
-  - [With React](#with-react)
-  - [With Fastify](#with-fastify)
-  - [With Hono](#with-hono)
-  - [With Express](#with-express)
-- [Using our amazing service](#using-our-amazing-service)
-- [Themes](#themes)
-- [Advanced: Styling](#advanced-styling)
-- [Community](#community)
-- [Other packages](#other-packages)
-- [Contributing](#contributing)
-- [License](#license)
+- [Scalar API Reference](#scalar-api-reference)
```

Comment:
let's remove this top level list item since the "Other Packages" and "Community" aren't a subset of this :)